### PR TITLE
Fixes Duplicate TC from Ambitions Module 

### PR DESCRIPTION
**Pull request by MissFox0810**
## About The Pull Request
Fixes a bug with the Ambitions module.
If a Antag as submitted and approved ambitions. then they drop there PDA, Then change objectives. give_antag is called again giving them another pda and bunch of TC.

Added check to that function to check if they have already been given the antag gear, and if so. just do nothing.
I think this works, but Needs checking by Azarak

## Changelog
:cl: Miss Fox
fix: Fixes to Ambitions module causing duplicate TC
/:cl:

### CODE REVIEW

File: code/modules/antagonists/traitor/datum_traitor.dm [hunk 1]
```diff
@@ -259,8 +261,9 @@
 	killer.add_malf_picker()
 
 /datum/antagonist/traitor/proc/equip(silent = FALSE)
-	if(traitor_kind == TRAITOR_HUMAN)
+	if(traitor_kind == TRAITOR_HUMAN && !/var/has_been_antag_geared)
```
Comment (NotRanged) [inline code/modules/antagonists/traitor/datum_traitor.dm]: should just be !has_been_antag_geared

File: code/modules/antagonists/traitor/datum_traitor.dm [hunk 2]
```diff
@@ -259,8 +261,9 @@
...
 		owner.equip_traitor(employer, silent, src)
+		/var/has_been_antag_geared = TRUE
```
Comment (NotRanged) [inline code/modules/antagonists/traitor/datum_traitor.dm]: same here

**Comment by Azarak** [comment]
ambitions_add() shouldn't ever be called twice by a single antag datum

**Comment by Azarak** [comment]
https://github.com/Skyrat-SS13/Skyrat-tg/pull/2421
is at fault as ambitions are designed to only be submitted once

**Comment by MissFox0810** [comment]
~~It does. We tested
Go antag. Submit your ambitions, (You get your pda), approve them.. Then drop your pda. and change your ambitions. you get another pda~~

**Comment by MissFox0810** [comment]
Ooooh

**Pull request closed without merging**